flocker/volume/schema/types.yml

```diff
@@ -0,0 +1,9 @@
+$schema: http://json-schema.org/draft-04/schema#
```

Comment:
I suspect types is going to get pretty big pretty soon. So maybe only move stuff there that is going to appear in more than one endpoint, and inline one-off types?